package.json

```diff
@@ -3,7 +3,8 @@
   "version": "4.3.1",
   "description": "A Photobooth webinterface for Linux and Windows",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.17.0 <19.0.0",

```

Comment:
Will the build process fail on v19 and above?
v16 is required by the build dependencies, on versions below v16 we're unable to create all needed resources while running `npm run build`.

Besides that node.js is needed only needed for the optional
- remotebuzzer server
- USB sync

If `npm run build ` doesn't fail on versions above v18 I would stop/limit the build process only because of optional features.